spanner/opencensus/src/java/com/example/spanner/opencensus/GrpcMetricSample.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2021 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.spanner.opencensus;
+
+// [START spanner_opencensus_grpc_metric]
+// Imports the Google Cloud client library
+
+import com.google.cloud.spanner.DatabaseClient;
+import com.google.cloud.spanner.DatabaseId;
+import com.google.cloud.spanner.ResultSet;
+import com.google.cloud.spanner.Spanner;
+import com.google.cloud.spanner.SpannerOptions;
+import com.google.cloud.spanner.Statement;
+import io.opencensus.contrib.grpc.metrics.RpcViews;
+import io.opencensus.exporter.stats.stackdriver.StackdriverStatsExporter;
+
+/**
+ * This sample demonstrates how to record and export client round-trip latency using OpenCensus.
+ */
+public class GrpcMetricSample {
+
+  public static void main(String[] args) throws Exception {
+    if (args.length != 2) {
+      System.err.println("Usage: GrpcMetricSample <instance_id> <database_id>");
+      return;
+    }
+
+    SpannerOptions options = SpannerOptions.newBuilder().build();
+    Spanner spanner = options.getService();
+    String instanceId = args[0];
```

Comment:
Command line arguments are an anti-pattern in samples. The sample classes should include a public method that contains the logic of the sample, which is called by the main method. This makes it easier to test samples and run them from an IDE. See the example in [SAMPLE_FORMAT.md](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/master/SAMPLE_FORMAT.md) for more details on what sample structure should look like.

Reply:
Done, ptal